src/components/LoanCards/Buttons/LendButton.vue

```diff
@@ -69,9 +72,24 @@ export default {
 			}).catch(() => {
 				this.$showTipMsg('Failed to add loan. Please try again.', 'error');
 			}).finally(() => {
-				this.loading = false;
+				this.setLoading(false);
 			});
-		}
+		},
+		setLoading(isLoading) {
+			this.loading = isLoading;
+			this.$emit('update:loading', isLoading);
+		},
+		trackCash103() {
+			const lendIncrementExperimentVersion = this.apollo.readQuery({
+				query: experimentQuery,
+				variables: { id: 'lend_increment_button' },
+			});
+			// eslint-disable-next-line max-len
+			const lendIncrementExperimentVersionString = _get(lendIncrementExperimentVersion, 'experiment.version') || null;
+			if (lendIncrementExperimentVersionString === 'variant-b') {
+				this.$kvTrackEvent('lending', 'EXP-CASH-103-Jan2019-click-lendnow', this.price);
```

Comment:
Please make 'lending', 'Lending' to match the category already in use.

Reply:
Nice catch :+1: